Converter/igtlioImageConverter.cxx

```diff
@@ -286,7 +288,12 @@ int igtlioImageConverter::IGTLToVTKImageData(igtl::ImageMessage::Pointer imgMsg,
       }
 
     }
-  
+
+  // Mark scalars as modified
```

Comment:
Add to the comment that this is needed for example for getting up-to-date scalar range.